### PR TITLE
various editorial cleanups

### PR DESCRIPTION
**Pull request by equalsJeffH**
[ using same "jeffh-editorial-cleanups" branch name as before, please do not delete after merge ]

Various editorial cleanups, see commts below (from 15-Sep onwards) for which issues/things are cleaned up...

[ commits prior to 15-Sep are already merged-to-master.  unable to easily squash those commits due to interstitial merges-from-master.  Click on the "Commits" tab above to easily see the commits listed by date. ]

### CODE REVIEW

**Comment by equalsJeffH** [comment]
Am not ready to merge this yet -- may have more items to add to it as we work thru the WD-02 issues...

File: index.bs [hunk 1]
```diff
@@ -2297,7 +2298,7 @@ handled on the server side and do not need support from the API specified here.
 
 
 # Acknowledgements # {#acknowledgements}
-We thank the following for their contributions to, and thorough review of, this specification: Jing Jin and Giridhar Mandyam.
+We thank the following for their contributions to, and thorough review of, this specification: Domenic Denicola, Brad Hill, Jing Jin, Anne van Kesteren, Giridhar Mandyam, Axel Nennker, Yaron Sheffer.
```
Comment (vijaybh) [inline index.bs]: Would you please also add Mike West and Anne van Kesteren?

Comment (equalsJeffH) [inline index.bs]: doh!  of course...

Comment (equalsJeffH) [inline index.bs]: I have Anne in there already -- adding Mikey.

Comment (equalsJeffH) [inline index.bs]: also adding Rahul Ghosh.

**Pull request merged**